src/betterproto/templates/template.py.j2

```diff
@@ -1,13 +1,13 @@
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
-# sources: {{ ', '.join(description.files) }}
+# sources: {{ ', '.join(description.input_filenames) }}
```

Comment:
Just for sanity sake it would be great to see the changes to names etc (and the template as a whole) to be split out into another PR just so that in case of any downstream issues we can revert easily. Just food for thought.

Reply:
I kind of agree. There's two main reasons why names were changed:
(1) When using the old names in the new structure really didn't make sense (ex: `properties` being a container for `fields`).
(2) When I just renamed things as I was developing for the sake of keeping my sanity and understanding everything.

I'd say the former makes sense for this PR but the latter (along with other re-naming for clarity's sake) would have made sense for another PR. The issue is that I don't remember which is which at this point 😣